<?xml version="1.0" ?>
<testsuite errors="2" failures="0" name="Tests.RCR_VR_HA_Service_ConfigTool_Config_Items_PUT.TestSuiteConfigItemsPut-20180712170712" tests="4" time="19.748">
	<testcase classname="Tests.RCR_VR_HA_Service_ConfigTool_Config_Items_PUT.TestSuiteConfigItemsPut" name="test_Step001_postRequest_new_id" time="5.455"/>
	<testcase classname="Tests.RCR_VR_HA_Service_ConfigTool_Config_Items_PUT.TestSuiteConfigItemsPut" name="test_Step002_putRequest_invalid_config_id" time="5.066"/>
	<testcase classname="Tests.RCR_VR_HA_Service_ConfigTool_Config_Items_PUT.TestSuiteConfigItemsPut" name="test_Step003_putRequest_empty_key_value" time="4.732">
		<error message="'int' object is not subscriptable" type="TypeError">
<![CDATA[Traceback (most recent call last):
  File "C:\Users\SC057441\Desktop\Any\pythonscripts\tokka\yamlgenerator\Requests\Final_Suite\Tests\RCR_VR_HA_Service_ConfigTool_Config_Items_PUT.py", line 67, in test_Step003_putRequest_empty_key_value
    completeURL = str(self.GET_URL) + "/" + str(self.id_table[0])
TypeError: 'int' object is not subscriptable
]]>		</error>
	</testcase>
	<testcase classname="Tests.RCR_VR_HA_Service_ConfigTool_Config_Items_PUT.TestSuiteConfigItemsPut" name="test_Step004_putRequest_empty_clientid" time="4.495">
		<error message="list index out of range" type="IndexError">
<![CDATA[Traceback (most recent call last):
  File "C:\Users\SC057441\Desktop\Any\pythonscripts\tokka\yamlgenerator\Requests\Final_Suite\Tests\RCR_VR_HA_Service_ConfigTool_Config_Items_PUT.py", line 57, in test_Step004_putRequest_empty_clientid
    self.id_table = self.compare_against_DB(params["client_id"],1)
  File "C:\Users\SC057441\Desktop\Any\pythonscripts\tokka\yamlgenerator\Requests\Final_Suite\Tests\RCR_VR_HA_Service_ConfigTool_Config_Items_PUT.py", line 21, in compare_against_DB
    results = sel_results[0]
IndexError: list index out of range
]]>		</error>
	</testcase>
	<system-out>
<![CDATA[java -jar Tests\auth-header-1.3.jar -k 99bb308b-0a2b-4547-a967-b6287914469d -s cHowMqwBMh2N4ShhBDrbcRZNLzo76VrA -p https://api.devcernercare.com/oauth/access > oAuthKey.txt
java -jar Tests\auth-header-1.3.jar -k 99bb308b-0a2b-4547-a967-b6287914469d -s cHowMqwBMh2N4ShhBDrbcRZNLzo76VrA -p https://api.devcernercare.com/oauth/access > oAuthKey.txt
java -jar Tests\auth-header-1.3.jar -k 99bb308b-0a2b-4547-a967-b6287914469d -s cHowMqwBMh2N4ShhBDrbcRZNLzo76VrA -p https://api.devcernercare.com/oauth/access > oAuthKey.txt
(729, 'test_blr', 'test', 'test', datetime.datetime(2018, 7, 12, 11, 35, 49), datetime.datetime(2018, 7, 12, 11, 35, 49), 0)
java -jar Tests\auth-header-1.3.jar -k 99bb308b-0a2b-4547-a967-b6287914469d -s cHowMqwBMh2N4ShhBDrbcRZNLzo76VrA -p https://api.devcernercare.com/oauth/access > oAuthKey.txt
]]>	</system-out>
	<system-err>
<![CDATA[]]>	</system-err>
</testsuite>
